sdk/contentsafety/Azure.AI.ContentSafety/api/Azure.AI.ContentSafety.netstandard2.0.cs

```diff
@@ -1,119 +1,161 @@
 namespace Azure.AI.ContentSafety
 {
-    public partial class AddBlockItemsOptions
+    public partial class AddOrUpdateTextBlocklistItemsOptions
```

Comment:
The api seems not correct. Have you called `eng\scripts\Export-API.ps1 contentsafety` to update?